src/main/java/com/google/devtools/build/lib/remote/blobstore/CombinedDiskHttpBlobStore.java

```diff
@@ -52,10 +59,10 @@ public boolean containsKey(String key) {
 
   @Override
   public ListenableFuture<Boolean> get(String key, OutputStream out) {
```

Comment:
This part is intact - it's still the same logic with the only difference - it is called explicitly from the `getActionResult` method